tests/cases/text/cursor_move.slint

```diff
@@ -14,17 +14,24 @@ TestCase := TextInput {
 /*
 ```rust
 
+const UP_CODE: char = '\u{F700}';
+const DOWN_CODE: char = '\u{F701}';
 const LEFT_CODE: char = '\u{F702}';
 const RIGHT_CODE: char = '\u{F703}';
-const HOME_CODE: char = '\u{F729}';
-const END_CODE: char = '\u{F72B}';
 const BACK_CODE: char = '\u{0008}'; // backspace \b
 
 let shift_modifier = slint::re_exports::KeyboardModifiers {
     shift: true,
     ..Default::default()
 };
 
+let move_mod_shift_modifier = slint::re_exports::KeyboardModifiers {
+    shift: true,
+    control: if cfg!(not(target_os = "macos")) { true } else {false},
+    alt: if cfg!(target_os = "macos") { true } else {false},
```

Comment:
```suggestion
    control: if cfg!(not(target_os = "macos")) { true } else { false },
    alt: if cfg!(target_os = "macos") { true } else { false },
```

Reply:
whoops. fixed now by amend and force push.

Reply:
actually the whole if-else is useless. I'll remove that as well.